kvdb/threads/pool_test.go

```diff
@@ -0,0 +1,39 @@
+package threads
+
+import (
+	"runtime/debug"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMain(m *testing.M) {
+	debug.SetMaxThreads(10)
+}
+
+func TestThreadPool(t *testing.T) {
+	require := require.New(t)
+
+	for name, pool := range map[string]ThreadPool{
+		"global": GlobalPool,
+		"local":  ThreadPool{},
+	} {
+		t.Run(name, func(t *testing.T) {
```

Comment:
@rus-alex I try to run but this test seems not running then it always passed with all inputs. I try to set the line #22 like this `require.Equal(pool.Cap(), 100)` but still no test failed.

And if we set the max threads to 10 then the pool cap should be 8, right?

Reply:
@hadv , thanks a lot, I forgot enable testing in TestMain()